go/engine/wot_test.go

```diff
@@ -113,12 +113,15 @@ func TestWebOfTrustPending(t *testing.T) {
 	require.NoError(tcAlice.T, err)
 	t.Log("alice and bob follow each other")
 
+	aliceName := alice.User.GetName()
+	bobName := bob.User.GetName()
+
 	var vouches []keybase1.WotVouch
-	vouches, err = libkb.FetchMyWot(mctxA)
+	vouches, err = libkb.FetchWotVouches(mctxA, libkb.FetchWotVouchesArg{Vouchee: nil, Voucher: nil})
```

Comment:
shouldn't need to specify the nil values
```suggestion
	vouches, err = libkb.FetchWotVouches(mctxA, libkb.FetchWotVouchesArg{})
```